plugin/storage/es/spanstore/reader.go

```diff
@@ -86,12 +86,12 @@ type SpanReader struct {
 	// this will be rounded down to UTC 00:00 of that day.
 	maxSpanAge                    time.Duration
 	serviceOperationStorage       *ServiceOperationStorage
-	spanIndexPrefix               string
-	serviceIndexPrefix            string
 	spanIndexDateLayout           string
 	serviceIndexDateLayout        string
 	spanIndexRolloverFrequency    time.Duration
 	serviceIndexRolloverFrequency time.Duration
```

Comment:
it would be better to just reuse IndexOptions struct here
```suggestion
	spanIndex   cfg.IndexOptions
	serviceIndex cfg.IndexOptions
```